src/main/kotlin/no/nav/familie/baks/mottak/søknad/kontantstøtte/KontantstøtteSøknadService.kt

```diff
@@ -69,6 +69,9 @@ class KontantstøtteSøknadService(
 
     fun hentDBKontantstøtteSøknad(søknadId: Long): DBKontantstøtteSøknad? = kontantstøtteSøknadRepository.hentSøknad(søknadId)
 
+    fun hentDBKontantstøtteSøknadForJournalpost(journalpostId: String): DBKontantstøtteSøknad? =
```

Comment:
Denne burde egentlig omdøpes til finn istedenfor hent, da det er mulig at den returnerer null (find vs get)

Reply:
Også kunne man hatt en annen metode som heter `hentDBKontantstøtteSøknadForJournalpost`, som kaller på find metoden og kaster feil hvis det ikke finnes. Så slipper man å bruke denne kodesnutten overalt man trenger det:

```
        if (søknad == null) {
            logger.error("Fant ikke søknad for journalpost=$journalpostId")
            throw IllegalStateException("Fant ikke søknad for journalpost=$journalpostId")
        }
```

Reply:
Fixed! Skrev også tester for metodene.